paradire/setup_patient_browser.sh

```diff
@@ -1,52 +1,57 @@
 #!/bin/bash
 set -e
 
-# Note, FHIR server must be running and available on localhost:8080 prior to running
-# this script.
+# Constants
+DOCKER_IMG_NAME="patient-browser"
+DOCKER_CONTAINER_NAME="patient-browser"
+DOCKER_PORT=9090
 
+# Note: Ensure FHIR server is running on localhost:8080 prior to running this script.
 curr_dir=$(pwd)
 
-has_node=$(which node)
-
-if [ -z "$has_node" ]; then
-    sudo apt install nodejs -y
+# Check for "patient-browser" directory and clone if missing
+if [ ! -d "$HOME/patient-browser" ]; then
+    git clone https://github.com/smart-on-fhir/patient-browser.git "$HOME/patient-browser"
 fi
 
-cd 
+cp "$curr_dir/patient_browser_patch/"* "$HOME/patient-browser"
 
-if [ ! -d "patient-browser" ]; then
-    sudo npm install typescript -g
-    git clone https://github.com/smart-on-fhir/patient-browser.git
-    ## Install dependencies and build project
-    (cd patient-browser && NODE_ENV=production npm install colors request commander@2.15.1 && npm ci && NODE_ENV=production npm run build)
-fi
+# Navigate to the "patient-browser" directory
+cd "$HOME/patient-browser"
 
-cd patient-browser
+# Create a Dockerfile
+cat <<EOL > Dockerfile
```

Comment:
Didn't we say we'd move this to a actual `Dockerfile` instead, in the patient_browser_patch dir?